sherpa.spec

```diff
@@ -5,6 +5,10 @@
 Source: git+https://github.com/%github_user/%{n}.git?obj=%{branch}/%{tag}&export=%{n}-%{realversion}&output=/%{n}-%{realversion}-%{tag}.tgz
 Requires: hepmc lhapdf blackhat sqlite fastjet openssl
 
+%if "%(case %cmsplatf in (*amd64*) echo true ;; (*) echo false ;; esac)" == "true"
```

Comment:
OS X is also _amd64_ . It needs to be slc*.

-- 
Ciao,
Giulio

On 19 Jan 2015, at 15:54, davidlt wrote:

>> @@ -5,6 +5,10 @@
>> Source: 
>> git+https://github.com/%github_user/%{n}.git?obj=%{branch}/%{tag}&export=%{n}-%{realversion}&output=/%{n}-%{realversion}-%{tag}.tgz
>> Requires: hepmc lhapdf blackhat sqlite fastjet openssl
>>
>> +%if "%(case %cmsplatf in (*amd64*) echo true ;; (*) echo false ;; 
>> esac)" == "true"
>
> `*_amd64_*` is more correct approach IMHO. This will not work on 
> AArch64 until compiler and linker improve. This should build on any 
> amd64 target, doesn't matter what linux distribution (for example at 
> some point I was building CMSSW on Fedora 20, ie f20_amd64_*). There 
> is nothing SLC specific here.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/cms-sw/cmsdist/pull/1295/files#r23163839